grpc/testing/test.proto

```diff
@@ -95,6 +95,7 @@ service LoadBalancerStatsService {
 // with HookRequestCommand::START
```

Comment:
Since the HookService serves both the `Hook` calls and the command call (like `ReturnAll`) there is no connection with the HealthService any more. You should update the comment on line 94

Reply:
Done